.circleci/config.yml

```diff
@@ -50,6 +50,14 @@ jobs:
             pip3 install --no-cache-dir -r dev-requirements.txt
           name: Install dependencies
 
+      - run:
+          name: fix singleuser base image for aarch64
+          # Jupyter docker-stacks don't have multi-arch images
+          # see https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/2844
+          command: |
+            sed -i -E s@base-notebook:latest@base-notebook:aarch64-latest@ images/singleuser-sample/Dockerfile
+            git diff
+
```

Comment:
```suggestion
```

With the new base image, we can be multi-platform and this is no longer relevant.